src/uv-common.c

```diff
@@ -225,6 +225,35 @@ int uv_ip6_name(const struct sockaddr_in6* src, char* dst, size_t size) {
   return uv_inet_ntop(AF_INET6, &src->sin6_addr, dst, size);
 }
 
+int uv_if_indextoname(unsigned int ifindex, char* ifname) {
+#ifdef _WIN32
+  NET_LUID InterfaceLuid;
+  int error = ConvertInterfaceIndexToLuid(ifindex, &InterfaceLuid);
+  if (NO_ERROR != error) {
+    return -error;
```

Comment:
I tried to have a look at the other Windows functions.  I hope I did the right thing.